src/psyclone/domain/gocean/nodes/gocean_extract_node.py

```diff
@@ -53,36 +54,7 @@ class GOceanExtractNode(ExtractNode):
     It adds a function 'generate_driver' which creates a GOcean-specific
     stand-alone driver program that can read the created output files.
```

Comment:
This begs the question: do we need to subclass ExtractNode if we're not adding any functionality to it? I would suggest not but open the floor to @hiker and @rupertford for discussion.